### ISSUE TITLE
[🐛 Bug]: You need to enable JavaScript to run this app. in UI

### ISSUE THREAD

**Issue by Doofus100500** [SeleniumHQ/docker-selenium#2221]
### What happened?

After updating to Chart with version 0.29.x, the UI stopped loading.
Browser error: You need to enable JavaScript to run this app.
However, everything loads through PortForward.

### Command used to start Selenium Grid with Docker (or Kubernetes)

```shell
helm
```

### Relevant log output

```shell
<body>
  <noscript>You need to enable JavaScript to run this app.</noscript>
  <div id="root"></div>
  <script src="index.js"></script>
</body>

Navigated to https://grid.test.com/staging/ui
GET https://grid.test.com/staging/index.js net::ERR_ABORTED 404 (Not Found)    ui:16
GET https://grid.test.com/staging/manifest.json 404 (Not Found)         manifest.json:1
```

### Operating System

k8s

### Docker Selenium version (image tag)

4.19.1-20240402

### Selenium Grid chart version (chart version)

0.29.1

**Comment by VietND96** [comment]
I see this is similar to an issue https://github.com/SeleniumHQ/selenium/issues/13757
access /ui is not working, and it require /ui/
@diemol, how do you think about this?

**Comment by VietND96** [comment]
`/ui` was changed to `/ui/` in this change form `4.19.0` https://github.com/SeleniumHQ/selenium/commit/bbd59478af06ea060dac342e406434c54056eba3#diff-360388c710d883e15e1f0e7cd8b95c2fe6671d1d51ffcbe961bac0d07b56168aR53-R55

**Comment by diemol** [comment]
Yes, there was a change due to the upgrades needed in the JS stack. We will have a look and have a fix for 4.21.

If you want, close this issue and mark it as duplicated (as it was already reported in the one you linked above).

**Comment by VietND96** [comment]
Ok, so let us follow up this on upstream repo.
